raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -25,55 +25,60 @@ contract TokenNetwork is Utils {
     // Channel identifier is a uint256, incremented after each new channel
     mapping (uint256 => Channel) public channels;
 
+    // We keep the balance data in a separate mapping to allow channel data structures to be
+    // removed when settling the channel. If there are locked transfers, we need to store
+    // balance data to be able to unlock them.
+    // Key is a hash of the channel_identifier and participant address.
+    mapping(bytes32 => BalanceData) balance_data;
+
     // Used for determining the next channel identifier
     // Start from 1 instead of 0, otherwise the first channel will have an additional
     // 15000 gas cost than the rest
     uint256 public last_channel_index = 0;
 
-    // The ClosingRequest identfier must be THE SAME as the channel identifier (mapping key) from `channels`
-    mapping (uint256 => ClosingRequest) public closing_requests;
-
-    struct Participant
-    {
+    struct Participant {
         // Total amount of token transferred to this smart contract through the
         // `setDeposit` function, note that direct token transfer cannot be
         // tracked and will be burned.
-        uint256 deposit;
-
-        // The latest known merkle root of the pending hash-time locks, used to
-        // validate the withdrawn proofs.
-        bytes32 locksroot;
+        uint240 deposit;
```

Comment:
This was thought as an optimization for gas usage. The other two variables `initialized` and `is_closer` are packed with `deposit` into a single slot of 32 bytes

Reply:
Are you sure this optimizes for gas succesfully and is it worth the discrepancy of using a different `uint` type? How much gas does this save?

Reply:
opened https://github.com/raiden-network/raiden-contracts/issues/46
not part of this PR imo.

Reply:
Agreed but this PR already makes an optimization by introducing a `uint240` for deposits and a `uint248` for block numbers.

And just because I agree it would make more sense to move this change, as per your issue, out of this PR.